native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert-test.common.js

```diff
@@ -44,6 +48,11 @@ const commonTests = () => {
     it('should render correctly dismissable', () => {
       const tree = renderer.create(
         <BpkBannerAlert
+          type={ALERT_TYPES.NEUTRAL}
```

Comment:
not a deal breaker but neutral is missing in the other tests further down, we should have consistency in the tests files

Reply:
Done :)